build-tools/packages/build-tools/src/common/npmPackage.ts

```diff
@@ -351,7 +351,7 @@ export class Packages {
 	 */
 	public static loadGlob(
 		globPath: string,
-		group: MonoRepoKind,
+		group: string,
```

Comment:
```suggestion
		releaseGroup: string,
```

Reply:
@tylerbutler , I have question about this one, because our "releaseGroup" definition aren't just for mono repo.  it can just be a set of packages with a group name (e.g. tools), which I don't think are released together?

Reply:
I removed this function.  It is unused.